datahub-web-react/src/app/search/SearchResultList.tsx

```diff
@@ -69,6 +69,11 @@ export const SearchResultList = ({
     const entityRegistry = useEntityRegistry();
     const selectedEntityUrns = selectedEntities.map((entity) => entity.urn);
 
+    const onClickExploreAll = useCallback(() => {
+        analytics.event({ type: EventType.SearchResultsExploreAllClickEvent });
+        navigateToSearchUrl({ query: '*', history });
```

Comment:
I dropped the `page: 0` from this to be consistent with the others. The navigateToSearchUrl defaults to page=1 which appears to work the same as page=0. One less thing to think about.